packages/components/src/image_zoom_slider/index.jsx

```diff
@@ -0,0 +1,40 @@
+import { useContext, useRef } from 'react';
+import Slider from '../slider';
+import Context from '../image_zoom_provider/context';
+import { useStateControlledInput } from './hooks';
+
+const ImageZoomSlider = (props) => {
+  const {
+    defaultScale,
+    maxScale,
+    scale,
+    previewSize,
+    onAnchorZoom,
+  } = useContext(Context);
+  const ref = useRef(null);
+
+  useStateControlledInput(ref, scale);
+
+  const handleZoomSlider = (value) => {
+    const anchor = {
+      x: previewSize.width / 2,
+      y: previewSize.height / 2,
+    };
+    onAnchorZoom(value / scale, anchor);
+  };
+
+  return (
+    <Slider
```

Comment:
add a note to refactor Slider later: 
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-29771